src/consumer/component/component-loader.ts

```diff
@@ -143,6 +143,7 @@ export default class ComponentLoader {
       );
       addExtensionsAsDevDependencies(component);
       updateDependenciesVersions(this.consumer, component);
+      addExtensionsAsDevDependencies(component);
```

Comment:
why did you add this? is it by mistake?

Reply:
Not sure. Might have been in one of my conversations with David about the graph. Awaiting his response.